composer.json

```diff
@@ -47,7 +47,7 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "symfony/flex": true
+            "symfony/flex": false
```

Comment:
why have you disabled symfony flex?

Reply:
Since this is not an ultimate symfony project that needs recipes executed. Currently, running `composer install` executes all the recipes right in the `SwaggerBundle`'s root. I don't think it should be the case, since it only brings the disturbing experience to anyone who would like to contribute.